src/main/java/in/sakthi/service/DisplayMedicine.java

```diff
@@ -16,12 +21,12 @@
 		medicines.add("Ranitidine");
 	}
 
-	static void display() {
+	public static void display() {
 		System.out.println("The Medicine is: " + medicines);
 	}
 
 	public static List<String> getMedicines() {
 		return medicines;
 	}
```

Comment:
why you are having separate class for display? You can written the medicine in service class also